plugin.video.lacapitv/addon.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.lacapitv" name="La Capi TV" version="1.0.0" provider-name="bazza">
+	<requires>
+		<import addon="xbmc.python" version="3.0.0"/>
+		<import addon="script.module.t1mlib" version="4.0.0"/>
+		<import addon="script.module.requests" version="2.22.0"/>
+	</requires>
+	<extension point="xbmc.python.pluginsource" library="default.py">
+		<provides>video</provides>
+	</extension>
+	<extension point="xbmc.addon.metadata">
+		<platform>all</platform>
+		<summary lang="es_AR">Una televisión interespacial</summary>
+		<description lang="es_AR">Un servicio de streaming intergaláctico donde podés encontrar animaciones de La Capi entre tantas otras series y pequeños experimentos de animación.
+</description>
+		<license>GNU General Public License, v3</license>
```

Comment:
Please use a spdx identifier instead of a full string: GPL-3.0-or-later in this case